Jenkinsfile

```diff
@@ -37,31 +34,37 @@ pipeline{
 		stage('Main: Run SBML-Exporter'){
 			steps{
 				script{
-					sh "mkdir -p ${folder}"
+					sh "mkdir -p ${env.OUTPUT_FOLDER}"
 					withCredentials([usernamePassword(credentialsId: 'neo4jUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-						sh "java -Xmx${env.JAVA_MEM_MAX}m -jar target/sbml-exporter-jar-with-dependencies.jar --user $user --password $pass --output ./${folder} --verbose"
+						sh "java -Xmx${env.JAVA_MEM_MAX}m -jar target/sbml-exporter-jar-with-dependencies.jar --user $user --password $pass --output ./${env.OUTPUT_FOLDER} --verbose"
 					}
 				}
 			}
 		}
+		// Creates archive of SBML files that will be copied to downloads folder and stored on S3.
+		stage('Post: Generate SBML archives and move to downloads folder') {
+		    steps{
+		        script{
+				def releaseVersion = utils.getReleaseVersion()
+				def speciesSBMLArchive = "all_species.3.1.sbml.tgz"
+				def humanSBMLArchive = "homo_sapiens.3.1.sbml.tgz"
+				
+				sh "cd ${env.OUTPUT_FOLDER}; tar -zcf ${speciesSBMLArchive} R-*"
+				sh "cd ${env.OUTPUT_FOLDER}; tar -zcf ${humanSBMLArchive} R-HSA-*"
+				sh "mv ${env.OUTPUT_FOLDER}/*.sbml.tgz ."
+				// Copy files to downloads/XX folder.
+				sh "cp *.sbml.tgz ${env.ABS_DOWNLOAD_PATH}/${releaseVersion}/"
```

Comment:
Should this be a move instead of copy? Or is there some other step that cleans up the original files?

Reply:
Its copied because the files are used for archiving on S3

Reply:
So... at the end of it there is only one set of files on the server?